CONTRIBUTING.md

```diff
@@ -300,8 +300,7 @@ Typeshed includes `scripts/create_baseline_stubs.py`.
 It generates stubs automatically using a tool called
 [stubgen](https://mypy.readthedocs.io/en/latest/stubgen.html) that comes with mypy.
 
-To get started, fork typeshed, clone your fork, and then
-[create a virtualenv](#-or-create-a-local-development-environment).
+To get started, fork typeshed, clone your fork, and then [prepare your environment](#preparing-the-environment).
```

Comment:
If I understand correctly, CONTRIBUTING.md doesn't need to be changed in this PR. Can you revert it to the original version, so that GitHub won't notify people that CONTRIBUTING.md has changed?